packages/shared/src/components/post/BasePostContent.tsx

```diff
@@ -13,7 +13,12 @@ const Custom404 = dynamic(
 const GoBackHeaderMobile = dynamic(
   () =>
     import(/* webpackChunkName: "goBackHeaderMobile" */ './GoBackHeaderMobile'),
-  { ssr: false },
+  {
+    ssr: false,
+    loading: () => (
+      <div className="-mx-4 h-12 border-b border-border-subtlest-tertiary laptop:hidden" />
```

Comment:
We normally use ElementPlaceholder I think it has a prefered bg color etc.
Also TIL you could use loading state on dynamic import 🙈

Reply:
<img width="599" alt="Screenshot 2024-10-21 at 11 43 05 AM" src="https://github.com/user-attachments/assets/d635a80f-5e0b-4444-8953-f16af16df8f3">

The color doesn't really make sense in this case as what we just want is the empty space to be there, so would prefer to keep it like this. 

> Also TIL you could use loading state on dynamic import 🙈
Same for when I raised the PR :D